e2e/helper/verify/hedera.go

```diff
@@ -0,0 +1,579 @@
+	t.Helper()
+	afterHbarBalance := fetch.HederaAccountBalance(t, hederaClient, hederaClient.GetOperatorAccountID())
+
+	var beforeTransfer uint64
+	var afterTransfer uint64
+
+	if asset == constants.Hbar {
+		beforeTransfer = uint64(beforeHbarBalance.Hbars.AsTinybar())
+		afterTransfer = uint64(afterHbarBalance.Hbars.AsTinybar())
+	} else {
+		beforeTransfer = beforeHbarBalance.Tokens.Get(tokenId)
+		afterTransfer = afterHbarBalance.Tokens.Get(tokenId)
+	}
+
+	if afterTransfer-beforeTransfer != expectedReceiveAmount {
+		t.Fatalf("[%s] Expected %s balance after - [%d], but was [%d]. Expected to receive [%d], but was [%d]",
+			hederaClient.GetOperatorAccountID(), asset, beforeTransfer+expectedReceiveAmount, afterTransfer, expectedReceiveAmount, afterTransfer-beforeTransfer)
+	}
+}
+
+func AccountBalance(t *testing.T, hederaClient *hedera.Client, hederaID hedera.AccountID, expectedReceiveAmount uint64, beforeHbarBalance hedera.AccountBalance, asset string) {
+	t.Helper()
+	afterHbarBalance := fetch.HederaAccountBalance(t, hederaClient, hederaID)
+
+	tokenAsset, err := hedera.TokenIDFromString(asset)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	beforeTransfer := beforeHbarBalance.Tokens.Get(tokenAsset)
+	afterTransfer := afterHbarBalance.Tokens.Get(tokenAsset)
+
+	if afterTransfer-beforeTransfer != expectedReceiveAmount {
+		t.Fatalf("[%s] Expected %s balance after - [%d], but was [%d]. Expected to receive [%d], but was [%d]", hederaClient.GetOperatorAccountID(), asset, beforeTransfer+expectedReceiveAmount, afterTransfer, expectedReceiveAmount, afterTransfer-beforeTransfer)
+	}
+}
+
+func SubmittedScheduledTx(t *testing.T, hederaClient *hedera.Client, mirrorNodeClient *mirror_node.Client, members []hedera.AccountID, asset string, expectedTransfers []transaction.Transfer, now time.Time) (transactionID, scheduleID string) {
+	t.Helper()
+	receiverTransactionID, receiverScheduleID := ScheduledTx(t, hederaClient, mirrorNodeClient, hederaClient.GetOperatorAccountID(), asset, expectedTransfers, now)
+
+	membersTransactionID, membersScheduleID := MembersScheduledTxs(t, hederaClient, mirrorNodeClient, members, asset, expectedTransfers, now)
+
+	if receiverTransactionID != membersTransactionID {
+		t.Fatalf("Scheduled Transactions between members are different. Receiver [%s], Member [%s]", receiverTransactionID, membersTransactionID)
+	}
+
+	if receiverScheduleID != membersScheduleID {
+		t.Fatalf("Scheduled IDs between members are different. Receiver [%s], Member [%s]", receiverScheduleID, membersScheduleID)
+	}
+
+	return receiverTransactionID, receiverScheduleID
+}
+
+func ScheduledMintTx(t *testing.T, hederaClient *hedera.Client, mirrorNodeClient *mirror_node.Client, account hedera.AccountID, asset string, expectedTransfers []transaction.Transfer, now time.Time) (transactionID, scheduleID string) {
```

Comment:
I think that we can remove `hederaClient` - currently we are not using it in this funciton